src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -966,6 +999,13 @@ public LocalizationMap getNameLocalizations()
             return nameLocalizations;
         }
 
+        @Nonnull
+        @Override
+        public String getCommandPath()
+        {
+            return parentCommand.getCommandPath() + " " + getName();
```

Comment:
Can you change the `getCommandPath` in [CommandInteractionPayload](https://github.com/DV8FromTheWorld/JDA/blob/7b3ba8355d98ab82a479947a38138589d70f35d1/src/main/java/net/dv8tion/jda/api/interactions/commands/CommandInteractionPayload.java#L117) to be consistent with this